pkgs/racket-doc/openssl/openssl.scrbl

```diff
@@ -28,10 +28,10 @@ or with the Racket distribution. In particular:
 @filepath{libeay32.dll} and @filepath{ssleay32.dll}, which are
 included in the Racket distribution for Windows.}
 
-@item{For Mac OS X, @racketmodname[openssl] depends on
+@item{For macOS, @racketmodname[openssl] depends on
 @filepath{libssl.dylib} and @filepath{libcrypto.dylib}. Although those
-libraries are provided by Mac OS X 10.2 and later, their use is
-deprecated, so the Racket distribution for Mac OS X includes newer
+libraries are provided by macOS 10.2 and later, their use is
```

Comment:
Since the usage here is “10.2 __and later__”, I think it's better with “macOS”. If it were a specific reference to only the 10.2 release, then "OS X 10.2". Also, American corporations are the worst.